component-css/input.css

```diff
@@ -1,245 +1,26 @@
-    color:var(--banner-text-color);
-}
-
-.nav-items .badges:hover{
-    background-color: rgb(113 54 54 / 15%);
-    backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
-    border-top-left-radius: 1.5rem;
-    border-bottom-left-radius:1.5rem ;
-    color:var(--banner-text-color);
-}
-
-.nav-items .avtars:hover{
-    background-color: rgb(113 54 54 / 15%);
-    backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
-    border-top-left-radius: 1.5rem;
-    border-bottom-left-radius:1.5rem ;
-    color:var(--banner-text-color);
-}
-
-.nav-items .inputs:hover{
-    background-color: rgb(113 54 54 / 15%);
-    backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
-    border-top-left-radius: 1.5rem;
-    border-bottom-left-radius:1.5rem ;
-    color:var(--banner-text-color);
-}
-
-.nav-items .modals:hover{
-    background-color: rgb(113 54 54 / 15%);
-    backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
-    border-top-left-radius: 1.5rem;
-    border-bottom-left-radius:1.5rem ;
-    color:var(--banner-text-color);
-}
 
-.nav-items .cards:hover{
-    background-color: rgb(113 54 54 / 15%);
+.active{
+    background-color: rgb(156 148 148 / 15%);
     backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
     border-top-left-radius: 1.5rem;
     border-bottom-left-radius:1.5rem ;
     color:var(--banner-text-color) !important;
-
-}
-.copy{
-    position: relative;
-    width: 100%;
-    margin-top: 0.8;
-    margin-bottom: 0px;
-    border-top: 0.4px solid  #ad9aa3;
-}
-.footer{
-    position: relative;
-    bottom: 0px;
-    width: 100%;
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    padding: 0.9rem 2rem;
-}
-.designedBy{
-    color: var(--banner-text-color);
-    line-height: 1.3;   
-    font-family: var(--main-font);
-    font-weight: 600;
-    font-size: 0.8rem; 
-    text-align: left;
-}
-.footer a{
-    text-decoration: none;
-    color: var(--main-primary-color);
-}
- .name{
-    color: var(--main-primary-color);
-    font-weight: 600;
-}
-.footer .github a{
-    font-size: 1.1rem;
-}
-
-.active{
-    color: var(--banner-text-color) !important;
 }
 
 /*                             CONTENT BOX                    */
 
 
-.content-box{
-    margin-left:22.5rem ;
-    background: #FFFAFA;
-    height: 100%;
-    padding: 2.5rem;
-    font-family: var(--main-font);
-    font-weight: 600;
-    display: flex;
-    flex-direction: column;
-    justify-content: flex-start;
-    align-content: space-around;
-}
-
-.content-box h1,h2,h3,h4{
-   font-family: var(--main-font);
-   font-weight: 600;
-}
-
-
-.content-box .main-heading{
-    max-width: 100%;
-    font-size: 2.5rem;
-    color: var(--main-primary-color);
-
-}
 
 .content-box .introduction , .input-component-container {
     font-size: 1.55rem;
     padding: 1.2rem;
+    margin: 0.8rem;
+    border-radius: 10px;
+    background: #F5F6F7;
```

Comment:
I see you have been using this hex color code #F5F6F7 at many places. It would be better if you declare this as a variable inside root at the top of css file, so that it will be easy for you to reuse at multiple places.